troi/filters.py

```diff
@@ -190,3 +190,47 @@ def read(self, inputs, debug=False):
                 output.append(rec)
 
         return output
+
+
+class YearRangeFilterElement(troi.Element):
+    '''
+        Remove/keep recordings if they do or do not belong in a range of years
+    '''
+
+    def __init__(self, start_year, end_year, inverse=False):
```

Comment:
nice-to-have would be to make end_year optional so that you can say "Give me stuff from 1997" with a single argument, but not important.

Reply:
Now implemented.